src/terminal7.ts

```diff
@@ -1103,24 +1167,28 @@ export class Terminal7 {
     }
     saveDotfile(text) {
         this.cells.forEach(c => {
+            // @ts-ignore
             if (typeof(c.setTheme) == "function")
-                c.setTheme(this.conf.theme)
+                (c as unknown as Pane).setTheme(this.conf.theme)
         })
         terminal7.loadConf(TOML.parse(text))
         if (this.pb &&
-            ((this.pb.host != this.conf.net.peerbook) 
+            ((this.pb.host != this.conf.net.peerbook)
+                // TODO: is bug?
+                // @ts-ignore
              || (this.pb.peerName != this.conf.peerbook.peer_name)
              || (this.pb.insecure != this.conf.peerbook.insecure)
+                // @ts-ignore
              || (this.pb.email != this.conf.peerbook.email))) {
             this.pbClose()
             this.pb = null
-            this.pbConnect()
+            this.pbConnect().then()
```

Comment:
this looks silly.  I guess this function should be async and then we can `await` here. The function already returns a promise - https://capacitorjs.com/docs/apis/preferences#set

Reply:
This call is on `this.pbConnect()` not on `Preferences.set`. I added this call because my IDE was complaining about a promise being ignored. In the olden days this code would not work without a call to `then`, now it does not really matter. I'll remove it.